declarative-examples/buildPluginParentPOM.groovy

```diff
@@ -0,0 +1,29 @@
+pipeline {
+    tools {
+        maven "apache-maven-3.1.0"
+        jdk "default"
+    }
+
+    agent {
+        label "master"
```

Comment:
@HRMPW  Why remove  it?  It is clearer than putting `agent { label '...' }` in examples that don't need it.